### PR TITLE
chore: Remove Storybooks

### PR DESCRIPTION
**Pull request by icirellik**
This change removes storybook from force as a followup to this RFC: https://github.com/artsy/README/issues/334

There is some additional minor dependency cleanup that resulted as part of this PR:

* Several require modules that were indirectly referenced due to yarn deduplication have been added to the `package.json`
* `wait-on` was only used in a script and has been replaced by curl.
* `forever` was no longer used
* `loadable-components` is a duplicated of `@loadable/components`
* `react-stripe-elements` is a required dependency
* `immer` is a required dependency

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -83,6 +83,7 @@ COPY patches ./patches
 FROM yarn-base as yarn-deps
 
 RUN --mount=type=cache,target=/usr/local/share/.cache \
+  --mount=type=cache,target=/root/.cache/Cypress \
```
Comment (eessex) [inline Dockerfile:86]: Cypress changes look like scope creep for the storybooks issue, this should probably have their own PR.

File: package.json [hunk 2]
```diff
@@ -74,6 +73,7 @@
     "@artsy/stitch": "6.1.6",
     "@artsy/xapp": "1.0.6",
     "@loadable/component": "5.12.0",
+    "@loadable/server": "5.13.2",
```
Comment (eessex) [inline package.json:76]: What does this package bring? I don't see it required by any files, maybe also requires its own pr if unrelated to storybooks.

Comment (icirellik) [reply package.json:76]: https://github.com/artsy/force/blob/master/src/v2/Artsy/Router/buildServerApp.tsx#L31

File: Dockerfile (hunk 1 above)
Comment (icirellik) [reply Dockerfile:86]: Agreed! There are three separate changes in this PR.

* Removing storybook
* Removing wait-on (and a few extra packages)
* Cypress cache changes

I will create a PR for the Cypress changes and keep the package removal in this PR as separate commits.

File: src/v2/Apps/Artwork/Components/OtherWorks/RelatedWorksArtworkGrid.tsx [hunk 3]
```diff
@@ -165,31 +162,6 @@ export const RelatedWorksArtworkGridRefetchContainer = createRefetchContainer<
   `
 )
 
-// FIXME: Move to storybooks
```
[the remark this answers is not in the document]
Comment (icirellik) [reply src/v2/Apps/Artwork/Components/OtherWorks/RelatedWorksArtworkGrid.tsx:168]: I found no references to it in the code and everything still works. @damassi thoughts?

[the remark this answers is not in the document]
Comment (damassi) [reply src/v2/Apps/Artwork/Components/OtherWorks/RelatedWorksArtworkGrid.tsx:168]: We used to stitch this in when building out the new page, so def old 👍

**Review by damassi — APPROVED** [review]
glorious

**Review by eessex — APPROVED** [review]
